modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json

```diff
@@ -10060,14 +10061,524 @@
             }
           }
         ],
-        "description": "Validates a request appointing an organization as Power of Attorney (21-22).\n",
         "responses": {
           "200": {
-            "description": "Valid request response"
+            "description": "Valid request response",
+            "content": {
+              "application/json": {
+                "example": {
+                  "data": {
+                    "type": "form/21-22/validation",
+                    "attributes": {
+                      "status": "valid"
+                    }
+                  }
+                },
+                "schema": {
+                  "type": "object",
+                  "properties": [
+                    {
+                      "status": {
+                        "type": "string"
+                      }
+                    }
+                  ]
+                }
+              }
+            }
```

Comment:
I think the validate docs marged this morning. May want to update this branch. @acovrig